edge/pkg/devicetwin/dtcommon/util.go

```diff
@@ -220,3 +222,8 @@ func dataToAny(v interface{}) (*anypb.Any, error) {
 		return nil, fmt.Errorf("%v does not support converting to any", reflect.TypeOf(v))
 	}
 }
+
+// GetResourceID return resource ID
+func GetResourceID(namespace, name string) string {
+	return namespace + "/" + name
+}
```

Comment:
We already have another func `GetResourceID`, please remove them to pkg/util.go

Reply:
Done